bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcPanel.scala

```diff
@@ -388,6 +393,14 @@ private class AnalysisMethodPanel extends GridBagPanel {
     case ValueChanged(_)                => publish(new SelectionChanged(this))
   }
 
+  def update() = {
+    // OCSADV-345: Don't allow users to change sky aperture for NIRI, F2 and GSAOI, this functionality has not been verified for these instruments.
+    Option(owner.getContextInstrumentDataObject).foreach { _.getType match {
+      case InstNIRI.SP_TYPE | Flamingos2.SP_TYPE | Gsaoi.SP_TYPE  => sky.enabled = false; skyLabel.enabled = true; sky.peer.setValue(1.0)
+      case _                                                      => sky.enabled = true;
+    }}
+  }
```

Comment:
The code in this PR seems fine and this disabling for NIRI etc. is probably fine as well.  I'm always reluctant though to disable widgets when there's no clear (to the user) reason why and no mechanism for enabling them. Perhaps we could switch them to labels or set them invisible?

````scala
sky.visible = Option(...).exists { _.getType match {
  case InstNIRI.SP_TYPE | ... => false
  case _                      => true
}}
````

Speaking of enabled, all of these widgets are enabled even when the observation is observed, which I assume is okay since it doesn't really edit anything in the observation itself correct?